app/controllers/graphql_controller.rb

```diff
@@ -0,0 +1,50 @@
+class GraphqlController < ApplicationController
```

Comment:
VERY comforting to see GraphQL in the rails universe with a controller!

Reply:
i agree